### PR TITLE
feat: migrate to mui5

### LINKED ISSUE #153: Migrate to mui5

### PR DESCRIPTION
**Pull request by pyphilia**
close #153 

Will be merged before @alvrba PR.

### CODE REVIEW

File: cypress/integration/chatbox.spec.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ describe('Chatbox', () => {
     cy.get(`#${ITEM_CHATBOX_ID}`).should('be.visible');
   });
 
-  it('Side panel button should hide chatbox', () => {
+  it.only('Side panel button should hide chatbox', () => {
```
Comment (spaenleh) [inline cypress/integration/chatbox.spec.js:36]: remove ?

File: src/components/common/MainMenu.js [hunk 2]
```diff
@@ -46,6 +46,7 @@ const MainMenu = () => {
 
   return (
     <GraaspMainMenu id={MAIN_MENU_ID}>
+      <div style={{ height: 10 }} />
```
Comment (spaenleh) [inline src/components/common/MainMenu.js:49]: Is this to push the tree view bellow the app bar ? I think there is a component specifically for this.

File: src/components/common/SideContent.js [hunk 3]
```diff
@@ -26,31 +25,20 @@ import Chatbox from './Chatbox';
...
-  },
-  pinnedContainer: {
-    overflow: 'scroll',
-  },
-  content: {
+const StyledMain = styled('main')(({ theme, isShifted }) => {
+  const contentShift = isShifted
+    ? {
+        // // necessary for content to be below app bar
+        // ...theme.mixins.toolbar,
```
Comment (spaenleh) [inline src/components/common/SideContent.js:32]: I think we can remove this

File: src/components/main/Home.js [hunk 4]
```diff
@@ -59,11 +51,11 @@ const Home = () => {
 
     return (
       <>
-        <Divider className={classes.divider} />
+        <Divider mt={2} mb={2} />
```
Comment (spaenleh) [inline src/components/main/Home.js:54]: This can be shortened to `my={2}`

**Review by spaenleh — APPROVED** [review]
✨ Just a few comments, you can merge when done !

File: src/components/common/MainMenu.js (hunk 2 above)
Comment (pyphilia) [reply src/components/common/MainMenu.js:49]: It's actually just for the little margin... I don't have control on the inside and the outside of these two components... But I'll add a todo.

**Pull request merged**